public_html/common.php

```diff
@@ -67,10 +68,12 @@ function load_wiki_name($query_array) {
 	  'wiktionary' => 'wiktionary',
 	);
 	$short_project = $project_map[$project];
-	if ( $project === 'wikidata' ) {
-		$wiki_name = 'wikidatawiki';
```

Comment:
Keeping this will allow it to remain compatible with the current `replica.rb`. I'd rather start by deploying this update on the php side, and then doing the corresponding changes in the ruby app afterwards. (Then we could remove this, once all servers have switched to the updated ruby code.)